internal/operators/cnv/cnv_operator.go

```diff
@@ -141,16 +141,6 @@ func (o *operator) GenerateManifests(c *common.Cluster) (map[string][]byte, erro
 	return Manifests(c.Cluster.OpenshiftVersion)
 }
 
-// GetDisksRequirementForMaster provides a number of disks required in a master
```

Comment:
I think we can remove other Get*RequirementFor* like in other operators

Reply:
It will cause significant changes in the validation method and I wanted to refactor it separately, not to overload this PR. Are you OK with that?

Reply:
Yes, we can do it in another PR.